README.md

```diff
@@ -106,25 +106,30 @@ As it turns out, people usually find this document before they sign a contract w
 
 UPDATE after 2 years or so: nobody seems to actually track it, but now that we have more data, we know it takes approximately 90 minutes to set everything up, so in worst case we can compensate based on that value.
 
-### RT projects
+### RT / RTO projects
```

Comment:
I am not sure how to add a comment to a line that was not changed, so I will leave it here. You have "he" in a document a few times. I would change to something gender neutral, like "he/she"

Reply:
You can do that in a separate PR after I merge this one. Generally we try to not employ anyone who would be severely offended by a document that is not perfectly gender neutral, but I do agree that it would be better to do it as you say.